cross-doc-explainer.md

```diff
@@ -0,0 +1,114 @@
+# Contents
+
+# Introduction
+
+See [the main explainer](explainer.md) for detailed explanation about view transitions.
+This explainer extends the feature of same-document ("SPA") transitions to also include
+cross-document transitions ("MPA").
+
+While same-document transitions are useful in themselves and an important stepping stone, allowing
+cross-document transitions would unlock a capability that so far has been available only within the
+same document.
+
+# Goals
+
+## Compatible with same-document transitions
+
+The vast majority of work for cross-document transitions is based on same-document transitions.
+This allows developers to use the same techniques for both, and to avoid big refactors if
+changing architectures.
+
+## Declarative & Customizable
+
+Cross-document transitions should work by default without JavaScript intervention, and should have
+the right JavaScript knobs for when the defaults are not enough.
+
+# How it works
+
+## In a nutshell
+Documents declare that they support same-origin cross-document transitions, using a
+[meta tag](#opt-in). When a navigation between two such documents takes place (and has no
+cross-origin redirects in the middle), the state of the old document is captured, in the same
+manner as same-origin transitions. When the new document is ready for the transition (i.e., when all render-blocking resources are ready), the new state is captured,
```

Comment:
Just a nit about the phrasing "the new state is captured" which makes it sounds like it's a one-time snapshot. I think a more pedantic statement is that the new shared elements are discovered and their state is tracked for the duration of the animation

Reply:
Isn't it [capture the new state](https://drafts.csswg.org/css-view-transitions-1/#capture-new-state-algorithm) the SPA algorithm though?

Reply:
I made these references links to the "capture the old/new state" algorithms. If we want to rename them we can but I don't want to do that in the explainer if we don't change the spec.

IMO capture is OK though